.azure-pipelines/template-commonlib.yml

```diff
@@ -10,25 +10,32 @@ jobs:
       set -ex
       case $(Build.SourceBranchName) in
         202012 | 202106)
-          bldenv=buster
+          bldenvs="buster"
```

Comment:
Can we add a variable to indicate which distributions need to build? Not hardcode in the script?